t/unit/test_models.py

```diff
@@ -146,3 +148,12 @@ class ClockedScheduleTestCase(TestCase, TestDuplicatesMixin):
     def test_duplicate_schedules(self):
         kwargs = {'clocked_time': timezone.now()}
         self._test_duplicate_schedules(ClockedSchedule, kwargs)
+
+    # we must have a valid the time-zone (not UTC) to do a correct test 
+    @override_settings(TIME_ZONE='Africa/Cairo') 
+    def test_timezone_format(self):
+        """Make sure the scheduled time is not shown in UTC when time zone is used"""
+        tz_info             = pytz.timezone(settings.TIME_ZONE).localize(datetime.datetime.utcnow())
+        schedule, created   = ClockedSchedule.objects.get_or_create(clocked_time=tz_info)
+        self.assertEquals(str(schedule.clocked_time), str(schedule))    # testnig str(schedule) calls make_aware() internally
```

Comment:
also assert should be in pytest style for any celery project

Reply:
@auvipy 
when I run `pytest ` I get 97 errors, although when I run `pytest --fixtures ` it says collected 97 items.
any cluses?

Reply:
I get "fixture 'patching' not found"

Reply:
I have updated the assert and pushed a commit.
Please let me know if you need anything. 
Also, please let me know how to pytest it without errors.